spec/preservation_ingest/validate_moab_spec.rb

```diff
@@ -0,0 +1,43 @@
+# rubocop:disable Metrics/BlockLength
+describe Robots::SdrRepo::PreservationIngest::ValidateMoab do
```

Comment:
Can you do `RSpec.describe` so we don't have to use rspec monkeypatch?

Reply:
Fixed in other existing specs as well